index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <h1 id="title">Survey Form</h1>
+    <p id="description">A simple survey form</p>
+<div id="form">
+    <form action="" id="survey-form">
+      <label for="name" id="name-label">Enter your name</label>
+      <input
+        required
+        type="text"
+        name="name"
+        id="name"
+        placeholder="Full name"
+      />
+
+      <label for="email" id="email-label">Enter your E-mail</label>
+      <input
+        required
+        pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$"
+        type="email"
+        name="email"
+        id="email"
+        placeholder="adress@something.com"
+      />
+
+      <label for="number" id="number-label">Enter a number</label>
+      <input
+        type="text"
+        name="number"
+        id="number"
+        min="1"
+        max="10"
+        required
+        pattern="\d"
```

Comment:
This pattern check is not working

Reply:
Kindly, you should be more explicit when saying the error and suggest some way to correct it, it is also important to be friendly with your messages

Reply:
Thank you for the feedback, I updated the comment with your recommendations.

Reply:
Very good job following the suggestions